src/collaborative_platform/apps/help/static/help/tei_sample_document.xml

```diff
@@ -0,0 +1,156 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <?xml-model href="https://providedh.ehum.psnc.pl/tei_providedh.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+  <teiHeader>
+    <fileDesc>
+      <titleStmt>
+        <title>To make Pease Soop</title>
+      </titleStmt>
+      <publicationStmt>
+        <publisher>
+          <orgName>Des Zentrums für Gastrosophie</orgName>
+          <ptr target="http://gastrosophie.sbg.ac.at/"/>
+        </publisher>
+        <date when="">2015-02-11</date>
```

Comment:
I think that filling _when_ parameter here might be a good idea, the sample file should be as correct and abundant as possible.